### ISSUE TITLE
can't remove inna extension in safari

### ISSUE THREAD

**Issue by foxzzu** [iina/iina#2182]
**System and IINA version:**

- macOS 10.14.1
- IINA 1.0.0

**Expected behavior:**

Extension for safari can't be removed. When trying to remove the extension, a prompt appears: To uninstall "Open in IINA", you must first remove the "IINA" application.
**Actual behavior:**

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**

- [ ] MPV does not have this problem.

**How often does this happen?**

**Comment by foss-** [comment]
Frankly I find this an odd concept. So I have to uninstall IINA to be sure it is not doing something while I am using Safari? I really like IINA as a media player, but I don't want it interfering with my browser.

Isn't there a solution to this problem and why was the bug closed?

**Comment by saagarjha** [comment]
> So I have to uninstall IINA to be sure it is not doing something while I am using Safari?

No, by not checking the box to enable it the extension will not run. It cannot affect your browsing in any way.

**Comment by foss-** [comment]
This research seems to suggest otherwise: 
https://lapcatsoftware.com/articles/Safari-runs-disabled-extensions.html

**Comment by saagarjha** [comment]
This is a pretty serious lapse in judgement by the Safari and Product Security teams, and I’m disappointed in both for seemingly not understanding how PluginKit works. However, even in the face of this issue do not see IINA being able to fix what is essentially a platform bug. If it’s any consolation, IINA’s Safari App Extension is tightly sandboxed and its principal class has no custom initializer (which you can [confirm yourself](https://github.com/iina/iina/blob/develop/OpenInIINA/SafariExtensionHandler.swift)), meaning that it cannot do much even if is provided code execution by Safari.

**Comment by saagarjha** [comment]
https://lapcatsoftware.com/articles/Safari-runs-disabled-extensions2.html

FYI
